pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -103,7 +103,12 @@ func NewSigner(cl clientset.Interface, secrets informers.SecretInformer, configM
 		secretSynced:       secrets.Informer().HasSynced,
 		configMapLister:    configMaps.Lister(),
 		configMapSynced:    configMaps.Informer().HasSynced,
-		syncQueue:          workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "bootstrap_signer_queue"),
```

Comment:
Why didn't we add `NewTypedNamedRateLimitingQueue` that allows a similarly short notation?

Reply:
Because the NewNamedRatelimitingQueue constructor was marked as deprecated so I thought we don't want that